### PR TITLE
Added new images on pages with the old project creation flow

### PR DESCRIPTION
**Pull request by jonat123**
As a new project creation flow has been created, some images and GIFs were outdated.

I have now updated the image and GIF with the new flow instead.

### CODE REVIEW

**Review by sofietoft — COMMENTED** [review]
Looks good 💪 
Remember to remove the unused image as well.

I noticed that the steps outlined in this article might need to be updated as well: https://docs.umbraco.com/umbraco-cloud/~/revisions/P0IIVMUEyonaC5nfESfK/#umbraco-cloud-portal-project

**Comment by jonat123** [comment]
@sofietoft I've removed the images and old GIFs and changed the text to fit with the new project creation flow.

Let me know what you think :)

**Comment by sofietoft** [comment]
Suggestion:
Take out the Baseline option step from the general "Create a new project steps", and then perhaps create a note with the information instead.

Finally, the steps in the Baseline docs for creating a Child project needs to fit with the new Gif as well 💪

**Comment by jonat123** [comment]
@sofietoft I've updated the steps, let me know if it looks good :)

File: umbraco-cloud/getting-started/baselines/README.md [hunk 1]
```diff
@@ -28,24 +28,23 @@ To create a child project:
...
 {% endhint %}
 
-1. **Choose an Owner** from the drop-down list.
-2. In the **Technical Contact** section, enter your **Name**, **Email**, and **Telephone**.
-3. Select **I have read and agree to the terms and conditions and the Data Processing Agreement**.
-4. Click **Create Project**.
+7. **Choose an Owner** from the drop-down list.
+8. In the **Technical Contact** section, enter your **Name**, **Email**, and **Telephone**.
+9. Review the entered information and select **I have read and agree to the terms and conditions and the Data Processing Agreement**.
+10. Click **Create Project**.
```
Comment (sofietoft) [inline umbraco-cloud/getting-started/baselines/README.md:43]:
```suggestion
10. Review the entered information and select **I have read and agree to the terms and conditions and the Data Processing Agreement**.
11. Click **Create Project**.
```

**Review by sofietoft — COMMENTED** [review]
Rewrote some of the steps to ensure each list item start with a verb.
Also added a couple of missing steps.

**Comment by jonat123** [comment]
Merging in the PR.

**Pull request merged**